rocketpy/Environment.py

```diff
@@ -3518,7 +3523,7 @@ def geodesicToUtm(self, lat, lon, datum):
 
         return x, y, utmZone, utmLetter, hemis, EW
 
-    def utmToGeodesic(self, x, y, utmZone, hemis, datum):
+    def utmToGeodesic(self, x: float, y: float, utmZone: int, hemis: str, datum: str):
```

Comment:
```suggestion
    def utmToGeodesic(self, x, y, utmZone, hemis, datum):
```

Are we gonna start using function annotations now? I thought we had not decided on this yet

Reply:
I've inserted this for helping me and forgot to erase, my bad.
solved!